### ISSUE TITLE
Follower breaks when screen is rotated

### ISSUE THREAD

**Issue by corot** [turtlebot/turtlebot_android#7]
In fact, this is a know problem in all ros-android apps. We found a solution for panorama that turned to be quite complicated and awkward. Maybe is time to attack the problem with a proper solution?

When done, add a vertical layout, with video up and buttons down

**Comment by chadrockey** [comment]
@corot Yeah, I've worked around the rotation before, but it's VERY complicated.  Android loves to trash your state and make you rebuild on rotate.  Really difficult for ROS.

Here's my sample app to work this out, at least a little.
https://github.com/ros-android/android_robot_monitor/blob/master/src/org/ros/android/robot_monitor/MonitorApplication.java#L78

Please report any progress you make.

**Comment by kentsommer** [comment]
Using Fragments may be an option: http://www.androiddesignpatterns.com/2013/04/retaining-objects-across-config-changes.html
